quickstarts/csharp/PersonalizerExample/README.md

```diff
@@ -1 +1,8 @@
 # About this Quickstart
+This sample interactively takes the time of day and the users's taste preference as context, and sends it to the Azure Personalizer instance, which then returns the top personalized food choice, along with the recommendation probability distribution of each food item.
+The user then inputs whether or not Personalizer predicted correctly, which is data used to improve Personalizer's prediction model.
```

Comment:
Did this text come from somewhere? It's a little light but a good first start.

Reply:
Didn't come from anywhere I just wrote it from running the sample. Added more content and detailed instructions for this one and HttpRequestFeaturizer.